odoo_2_odoo_data_transfer/models/odoo_data_transfer_log.py

```diff
@@ -67,24 +61,58 @@ class OdooDataTransferTemplateLog(models.Model):
         domain=[("error_type", "=", "other_error")],
     )
     transfered_records_counter = fields.Integer(
-        compute="_compute_transfered_records_counter"
+        compute="_compute_transfered_records_counter",
+        store=True,
     )
     record_total_counter = fields.Integer()
 
+    first_transferred_id = fields.Integer(
+        compute="_compute_transfered_records_counter",
+        store=True,
+    )
+    last_transferred_id = fields.Integer(
+        compute="_compute_transfered_records_counter",
+        store=True,
+    )
+    is_failed_migration = fields.Boolean(readonly=True)
+
     @api.depends("transfered_records_ids")
     def _compute_transfered_records_counter(self):
         for rec in self:
             rec.transfered_records_counter = len(rec.transfered_records_ids)
+            rec.first_transferred_id = (
+                min(rec.transfered_records_ids.mapped("remote_id"))
+                if rec.transfered_records_ids
+                else 0
+            )
+            rec.last_transferred_id = (
+                max(rec.transfered_records_ids.mapped("remote_id"))
```

Comment:
If I'm not mistaken, it would be faster to perform a search with desc or asc order based on the remote_id field and a return limit of 1.